src/CustomVariableEditor/MeasurementFilter.tsx

```diff
@@ -37,9 +44,12 @@ export function MeasurementFilterRow(props: MeasurementFilterProps) {
           label={'Filter measurement'}
           aria-label={'Filter measurement'}
           labelWidth={20}
-          tooltip={<div>Searches measurement by name</div>}
+          tooltip={<div>Searches measurement by name, to use a regex surround pattern with /</div>}
         >
-          <Input value={props.initialValue?.Keyword} onChange={(e) => onKeywordChange(e)} />
+          <MaybeRegexInput
+            onChange={(value, ok) => onKeywordChange(value, ok)}
+            initialValue={props.initialValue?.Keyword}
+          />
```

Comment:
not sure anymore but I thought with the toggle that the '/' were not needed anymore here and that they were added automatically later. Might be misremembering though